ui/src/app/login/controllers/LoginCtrl.js

```diff
@@ -17,7 +18,15 @@
         } else if(status.state == 403) {
           $scope.incorrectCredentials = true;
         }
-       });
+      });
+    };
+
+    $scope.hideShowPassword = function() {
```

Comment:
```suggestion
    $scope.hideShowPassword = () => {
```
I think this will fix the codacy issues.